pkg/net/libp2p/channel_manager.go

```diff
@@ -114,3 +119,53 @@ func (cm *channelManager) newChannel(name string) (*channel, error) {
 
 	return channel, nil
 }
+
+func (cm *channelManager) newRelay(name string, ttl time.Duration) error {
+	cm.relaySubscriptionsMutex.Lock()
+	defer cm.relaySubscriptionsMutex.Unlock()
+
+	if _, ok := cm.relaySubscriptions[name]; !ok {
```

Comment:
What do you think about invalidating the previous relay and adding a new one? This way, we can increase a TTL. Right now, we may run into a situation when we should relay for the next hour but we stop one minute after the protocol started because the previous relayer was installed.

Reply:
I'm not sure. Subscribing to a topic is rather heavyweight and might take a similar amount of time as the signature shares exchanging process which is really quick now. Apart from that, we should look at this functionality as an auxiliary message relay. Even if it shutdowns during for example a relay entry (which is btw rather unlikely) nothing big happens.

Reply:
Maybe we can just set a higher value for relay entries? For example 24 hours or so?

Reply:
See it's not the problem here. The problem is that we install the forwarder and it's here for one hour. Even if we register another forwarder after 50 minutes, we are going to shutdown after 10 minutes.

Reply:
Yeah, I know but I tried to avoid making this solution complex because as I said this is just an auxiliary forwarder which is not essential for our protocols to work. If you think we should address this edge case I can obviously make this logic more sophisticated.

Reply:
We can also make forwarders for relay entry channels eternal. This is the simplest way to mitigate the problem we are discussing here.

Reply:
Let's leave it as-is and open an issue in `keep-core` referencing the issue from libp2p pubsub and also mention this problem there.

Reply:
Sure, let's do it.

Reply:
Created https://github.com/keep-network/keep-core/issues/1652